Latest depends tickling a bug in PIL

When checking a PR our CI reported that the development environments (the ones we let `pip upgrade`)  are tickling a very recent bug in PIL.

Both failed tests look like this (in APPLE, but this has nothing to do with APPLE).
```
>       im = np.array(Image.fromarray(im).resize(size, Image.BICUBIC), dtype=np.float64)
E       TypeError: __array__() takes 1 positional argument but 2 were given

../.tox/py36-dev/lib/python3.6/site-packages/aspire/apple/picking.py:151: TypeError
```

Confirmed it's this:
https://github.com/python-pillow/Pillow/issues/5571

I'll work around it (just requires moving the `dtype` arg to `astype(dtype)`). PR coming in a few minutes.

--- a/src/aspire/apple/picking.py
+++ b/src/aspire/apple/picking.py
@@ -148,7 +148,9 @@ class Picker:
         size = tuple((np.array(im.shape) / config.apple.mrc_shrink_factor).astype(int))
 
         # Note, float64 required for signal.correlate call accuracy.
-        im = np.array(Image.fromarray(im).resize(size, Image.BICUBIC), dtype=np.float64)
+        im = np.asarray(Image.fromarray(im).resize(size, Image.BICUBIC)).astype(
+            np.float64, copy=False
+        )
 
         im = signal.correlate(
             im,

